backend/src/impl/db_utils/system_db_utils.py

```diff
@@ -339,8 +326,18 @@ def _find_output_or_case_raw(
             filt=filt,
         )
         if total != 1:
-            abort_with_error_message(400, f"Could not find a single system {system_id}")
-        sys_data = SystemDBUtils._decompress_from_cursor(cursor)
+            abort_with_error_message(
+                400, f"Could not find system outputs for {system_id}"
+            )
+        data = next(cursor)["data"]
+        # NOTE: (backward compatibility) data was stored in MongoDB previously,
+        # this can be removed after we regenerate DB next time.
```

Comment:
Explaining what "this" is would make this more clear.

Reply:
I have updated the comment.